src/aliased_buffer.h

```diff
@@ -25,10 +25,7 @@ template <class NativeT, class V8T>
 class AliasedBuffer {
  public:
   AliasedBuffer(v8::Isolate* isolate, const size_t count)
-      : isolate_(isolate),
```

Comment:
i feel like this was more readable before

Reply:
This is `ConstructorInitializerAllOnOneLineOrOnePerLine`. There is apparently no option to force one per line.